### PR TITLE
Docker sync setup.

### PR DESCRIPTION
**Pull request by rasben**

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: "2"
+
+services:
+  ereol:
+    volumes:
+      - 'ereol-webroot-sync:/var/www/html'
+      - './private/docker/sites/ereol:/var/www/html/sites/default'
```
Comment (tripox) [inline docker-compose-dev.yml]: You should probably use docker-sync for this too.

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - './private/docker/init/prepare-environment.sh:/etc/my_init.d/prepare-environment.sh'
+  ego:
+    volumes:
+      - 'ego-webroot-sync:/var/www/html'
+      - './private/docker/sites/ego:/var/www/html/sites/default'
```
Comment (tripox) [inline docker-compose-dev.yml]: Same as above.

Comment (tripox) [inline docker-compose-dev.yml]: Same as above.

**Comment by xendk** [comment]
What about the *-gulp containers? Why are there two volumes, when it's one directory mounted into two containers?

**Pull request merged**